lib/utils/emitDeprecationWarning.mjs

```diff
@@ -19,8 +21,27 @@ const CODES = {
  * @param {string} detail
  * @returns {void}
  */
-export default function emitDeprecationWarning(message, codeKey, detail) {
+export function emitDeprecationWarning(message, codeKey, detail) {
```

Comment:
[suggestion] Is it possible to keep `export default` to prevent the increase of trivial changes?

```suggestion
export default function emitDeprecationWarning(message, codeKey, detail) {
```